tamr_unify_client/models/dataset/resource.py

```diff
@@ -60,8 +60,14 @@ def update_records(self, records):
         :param records: Each record should be formatted as specified in the `Public Docs for Dataset updates <https://docs.tamr.com/reference#modify-a-datasets-records>`_.
         :type records: iterable[dict]
```

Comment:
You should include a `:return:` and `:rtype:` in the docstring for `update_records` to reflect the new behavior. There are plenty of examples of this in the code base e.g. the `attributes` function just above `update_records`.